components/AccountHoverCard.tsx

```diff
@@ -314,20 +317,19 @@ export const AccountHoverCard = ({
         {trigger}
       </HoverCardTrigger>
       <HoverCardContent
-        className="w-80 cursor-default text-left text-sm"
+        className="w-80 cursor-default text-left text-sm font-normal"
         onClick={e => e.stopPropagation()} // Prevent click propagation when used inside other elements such as `HostedAccountFilter`
         {...hoverCardContentProps}
       >
         <div className="relative flex flex-col gap-4 text-sm">
-          <div
-            className={clsx(
-              'flex  gap-3 overflow-hidden break-words',
-              isIndividual ? 'flex-col items-start' : 'items-center',
-            )}
-          >
-            <Link href={getCollectivePageRoute(account)}>
-              <Avatar collective={account} radius={isIndividual ? 64 : 40} />
-            </Link>
+          <div className="flex flex-col gap-3 overflow-hidden break-words">
+            <div className="flex justify-between">
+              <Link href={getCollectivePageRoute(account)}>
+                <Avatar collective={account} radius={64} />
+              </Link>
+
+              {displayFollowButton && <FollowButton account={account} isHoverCard />}
```

Comment:
We shouldn't display the following button on our own profile. The same logic should be applied to the Collective page (if you have the time).

Reply:
Added a quick fix in the FollowButton to not display it if it is your own profile